### PR TITLE
DIV-2229-change-email-to-read-only-contact-details-master

### PR DESCRIPTION
**Pull request by niyioyelade**
'generic-error' is thrown when trying to save and close invalid email addresses on 'contact-details' - initial commit to branch

# Description
https://tools.hmcts.net/jira/browse/DIV-2229

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Screen changes done.
Email address on contact details page is now read-only
Contact detail page content is now as per screen design in the JIRA.
Removed email address change or validation codes.

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/assets/sass/_contact-details.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    background: #dee0e2;
+    padding: 25px;
+    .multiple-choice [type=checkbox] + label::before {
+      background: #fff;
+    }
+
+    .list-bullet {
+      padding-left: 70px;
+    }
+    .text {
```
Comment (dtaylor7) [inline app/assets/sass/_contact-details.scss]: cant see anywhere this class is being used?

File: app/assets/sass/_contact-details.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.contact-details {
+
+  .confirmation-container {
+    background: #dee0e2;
+    padding: 25px;
+    .multiple-choice [type=checkbox] + label::before {
+      background: #fff;
+    }
+
+    .list-bullet {
```
Comment (dtaylor7) [inline app/assets/sass/_contact-details.scss]: cant see anywhere this class is being used?

File: app/steps/petitioner/contact-details/content.json [hunk 3]
```diff
@@ -4,11 +4,13 @@
         "en": {
             "translation": {
                 "content": {
-                    "question": "Enter your contact details",
-                    "communicate": "You need to agree to receive notifications about your divorce and court documents by email.",
+                    "question": "How you'll be contacted",
+                    "communicateParagraph1": "You'll be sent emails by the court throughout the divorce.<br/> You must agree to receive these emails to use this service.",
```
Comment (dtaylor7) [inline app/steps/petitioner/contact-details/content.json]: you dont need / in `<br/>` here

File: app/steps/petitioner/contact-details/index.test.js [hunk 4]
```diff
@@ -107,18 +93,14 @@ describe(modulePath, () => {
 
   describe('success', () => {
     it('redirects to the next page when valid email is entered and consent is given', done => {
```
Comment (dtaylor7) [inline app/steps/petitioner/contact-details/index.test.js]: can you reword this test?

Comment (dtaylor7) [reply app/steps/petitioner/contact-details/index.test.js]: we could also do with a test to ensure the email address is rendered correctly on this page?

File: test/end-to-end/pages/petitioner/contact-details.js [hunk 5]
```diff
@@ -4,7 +4,6 @@ function enterPetitionerContactDetails() {
   const I = this;
 
   I.seeCurrentUrlEquals('/petitioner-respondent/contact-details');
-  I.fillField('petitionerEmail', 'example@example.com');
```
Comment (dtaylor7) [inline test/end-to-end/pages/petitioner/contact-details.js]: perhaps change to `I.see('example@example.com');`?

Comment (niyioyelade) [reply test/end-to-end/pages/petitioner/contact-details.js]: Done

**Comment by niyioyelade** [comment]
Merged with 2305 in a separate PR

**Pull request closed without merging**